lib/widgets/message_list.dart

```diff
@@ -459,30 +459,41 @@ class ScrollToBottomButton extends StatelessWidget {
   }
 }
 
-class MarkAsReadWidget extends StatelessWidget {
+class MarkAsReadWidget extends StatefulWidget {
   const MarkAsReadWidget({super.key, required this.narrow});
 
   final Narrow narrow;
 
+  @override
+  State<MarkAsReadWidget> createState() => _MarkAsReadWidgetState();
+}
+
+class _MarkAsReadWidgetState extends State<MarkAsReadWidget> {
+  @visibleForTesting
+  bool loading = false;
```

Comment:
```suggestion
class _MarkAsReadWidgetState extends State<MarkAsReadWidget> {
  bool loading = false;
```

Pretty sure this annotation has no effect when the class itself is private.